src/vm/ClrEtwAll.man

```diff
@@ -1415,12 +1437,12 @@
                     <template tid="ContentionStop_V1">
                         <data name="ContentionFlags" inType="win:UInt8" map="ContentionFlagsMap" />
                         <data name="ClrInstanceID" inType="win:UInt16" />
-                        <data name="Duration" inType="win:Double" />
+                        <data name="DurationNs" inType="win:Double" />
```

Comment:
Is this change intentionally added to this PR?

Reply:
Yea just a bit of cleanup for a recently added field